must-gather/must-gather-requirements.md

```diff
@@ -1,60 +1,53 @@
-# Requirements for OpenShift sandboxed containers must-gather
+# <center>Requirements for OpenShift Sandboxed Containers (OSC) must-gather</center>
 
-`must-gather` for OpenShift sandboxed contains (OSC) should gather all information and logs needed for debugging OSC.
 
 ### Usage
+The kataconfig must have `logLevel: debug` set before running `must-gather`.
+
+OSC `must-gather` should gather all OCS information and logs needed for debugging in a directory
 ```sh
 oc adm must-gather --image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest
 ```
+Data about other parts of the cluster is gathered with `oc adm must-gather`. Run `oc adm must-gather -h` to see more options.
 
-The command above will create a local directory with a dump of the OpenShift sandboxed-containers state.
-Note that this command will only get data related to the sandboxed-containers part of the OpenShift cluster.
-
-### Contents of the dump
-- All namespaces (and their children objects) that belong to any sandboxed containers resources
-
-In order to get data about other parts of the cluster (not specific to sandboxed containers) you should
-run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+### Openshift Sandboxed Containers
+Kata runtime is the `containerd-shim-kata-v2` process that talks to the lata agent in the VM.
+See also the [Official 1.5 documentation](https://access.redhat.com/documentation/en-us/openshift_sandboxed_containers/1.5/html-single/openshift_sandboxed_containers_user_guide/index#troubleshooting-sandboxed-containers)
 
-#### General
+#### Gathered Data
 - Resource definitions
 - Service logs
+- All namespaces and child objects with OSC resources
+- All OSC custom resource definitions (CRDs)
+  - sandboxed-containers/namespaces/openshift-sandboxed-containers-operator/**_*_**\_description
+- virtiofsd
 
-#### OSC
-- All namespaces and their child objects that belong to any OpenShift sandboxed containers resources
-- All OpenShift sandboxed containers custom resource definitions (CRDs)
-  - sandboxed-containers/namespaces/openshift-sandboxed-containers-operator/**_*\_description_**
-
-- Kata agent logs **???????**
-- Kata runtime logs **??????**
 
-- QEMU logs
-  - part of the **crio** logs as subsystem=qemu and subsystem=qmp
-- Audit logs
+#### Locations
+- CRI-O logs - from the kata runtime
+  - nodes/**_nodename_**/**_nodename_**\_logs\_crio
+- QEMU
+  - logs are part of the **CRI-O** logs as _subsystem=qemu_ , _subsystem=qmp_ and/or _qemuPID=**PID**_
+  - version
+    - nodes/**_nodename_**/**_nodename_**/version
+- virtiofsd
+  - virtiofsd logs are part of the **CRI-O** logs
```

Comment:
@gkurz Would `oc adm node-logs "$NODE" -u "$UNIT"`  work?  That is how must-gather/gather_nodes gets `kublet` & `crio`

Reply:
@tbuskey No because `virtiofsd` isn't a systemd unit. Since the shim v2 spawns `virtiofsd` with the `--syslog` argument, one could expect to collect the logs with `oc adm node-logs "$NODE" -t virtiofsd`. This used to work with the C virtiofsd but it doesn't produce anything with the Rust virtiofsd... A possible workaround to avoid having to log into the node is to do something like `oc adm node-logs "$NODE" -g ' virtiofsd\[[[:digit:]]+\]: '`

Reply:
I modified gather_nodes to collect virtiofsd logs & tested